scrap/main.py

```
@@ -0,0 +1,52 @@
+from api import fetch_api
+from db import get_book_by_title_author, insert_book
+from datetime import datetime
+
+
+def normalize_title(title: str) -> str:
+    return title.replace(" ", "").lower()
+
+
+def process_query(query_char: str):
```

Ruff (0.11.9)

Missing return type annotation for public function `process_query`

Add return type annotation: `None`

(ANN201)